apps/accounts/serializers.py

```diff
@@ -69,3 +70,12 @@ def update(self, instance, validated_data):
             profile.linkedin_url = linkedin_url
             profile.save()
         return instance
+
+
+class CustomPasswordResetSerializer(PasswordResetSerializer):
+    def get_email_options(self):
```

Comment:
Add docstrings to explain what this serializer does. Refer [this](https://github.com/Cloud-CV/EvalAI/blob/master/apps/accounts/serializers.py#L7) for an example

Reply:
Done